packages/wonder-blocks-theming/src/hooks/use-styles.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import {StyleDeclaration, StyleSheet} from "aphrodite";
+
+import {ThemedStylesFn} from "../types";
+
+/**
+ * A hook that that applies styles based on a given theme.
+ *
+ * @param styles The styles to use.
```

Comment:
Would it be more accurate to say that this is a function that returns the styles to use?